61.md

```diff
@@ -0,0 +1,64 @@
+NIP-61
+======
+
+Event Sets
+----------
+
+`draft` `optional`
+
+An event set is a group of events with the same pubkey and `n` (set "n"ame) tag.
+An event can have many `n` tags, thus being member of many event sets.
+
+To add an event to the "Close Friends" set,
+add a "close-friend" (prefer lowercase English words in singular form with dash separator) `n` tag to the event.
+
+To remove an event from a "Close Friends" set,
+remove the "close-friend" `n` tag from the event or send a deletion event (`kind:5`).
+
+An event set itself is referenced by an `s` tag
+or by an `nset` [NIP-19](19.md) entity using [NIP-21](21.md) URI.
+
+## Referencing
+
+An event set is referenced by an `s` tag.
+It references a set of events from an author instead of a single one:
+
+`["s", "<kind, optional>:<32-bytes lowercase hex of a pubkey>:<n tag value>", "<recommended relay URL, optional>"]`
+
+One may also use an `nset` entity URI: `nostr:nset1qqstn...794234d`.
+
+## Event Kinds
+
+The same `n` tag can be applied to any event no matter the `kind` or structure.
+
+The following example is based on the unreleased
+[Relationship Status](https://github.com/vitorpamplona/nips/blob/relationship-status/81.md) spec:
+
+```js
+{
+  "kind": 30382, // This is a kind for public relationship info between the signer and another user
+  "pubkey": "<me>",
+  "tags": [
+    ["d", "<another-user-pubkey>"],
```

Comment:
What possible values can go in the `d` tag? That's what makes this whole idea valuable, right? I feel this NIP is under-spec'd without it.

Reply:
You can see more on #761. This PR is just for the idea of `n` tags to represent sets of events. It uses the status as an example, but it doesn't define it. Overthere, we predicted either the d=pubkey of the "friend" OR a private SHA256 of the key for use cases where the relationship itself is private.

Reply:
Are the ideas really separable? I'm having trouble picturing what other kinds of event sets would look like, to justify the `d` tag being free-form.

Reply:
I can use `n` tags to put spreadsheets into lists (or directories) :)

Reply:
Or group my Classifieds into collections.